README.md

```diff
@@ -112,6 +113,15 @@ Command to exit the file writer:
 {"cmd": "FileWriter_exit"}
 ```
 
+Command to stop a single file:
+
+```json
+{
+	"cmd": "FileWriter_stop",
+	"jobid": "job-unique-identifier"
+}
+```
+
```

Comment:
Should we provide an optional `stop_time` here?

Reply:
:white_check_mark: